internal/anchor/reconciler_test.go

```diff
@@ -2,15 +2,13 @@ package anchor_test
 
 import (
 	"context"
-	"testing"
-
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"k8s.io/apimachinery/pkg/types"
-
 	api "sigs.k8s.io/hierarchical-namespaces/api/v1alpha2"
 	"sigs.k8s.io/hierarchical-namespaces/internal/config"
 	. "sigs.k8s.io/hierarchical-namespaces/internal/integtest"
+	"testing"
```

Comment:
Can you please restore this part? All our Go files have three "paragraphs" of imports: the standard library, then external packages, then other HNC packages.